ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -50,55 +50,59 @@ open class Tooltip: NSObject, TokenizedControlInternal {
                                                            arrowMargin: tokenSet[.backgroundCornerRadius].float,
                                                            tokenSet: tokenSet
         )
-        let tooltipView = TooltipView(message: message,
-                                      title: title,
-                                      textAlignment: textAlignment,
-                                      positionController: positionController,
-                                      tokenSet: tokenSet)
-        self.tooltipView = tooltipView
-        tooltipView.accessibilityViewIsModal = true
-
-        self.onTap = onTap
-
-        self.dismissMode = UIAccessibility.isVoiceOverRunning ? .tapOnTooltip : dismissMode
-
-        let gestureView = TouchForwardingView(frame: window.bounds)
-        self.gestureView = gestureView
-        switch self.dismissMode {
-        case .tapAnywhere:
-            window.addSubview(tooltipView)
-            window.addSubview(gestureView)
-            gestureView.onTouches = { _ in
-                self.handleTapGesture()
-            }
-        case .tapOnTooltip, .tapOnTooltipOrAnchor:
-            window.addSubview(gestureView)
-            window.addSubview(tooltipView)
-            gestureView.forwardsTouches = false
-            tooltipView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGesture)))
-            if self.dismissMode == .tapOnTooltipOrAnchor {
-                gestureView.passthroughView = anchorView
-                gestureView.onPassthroughViewTouches = { _ in
+        self.tooltipViewController = TooltipViewController(message: message,
+                                                           title: title,
+                                                           textAlignment: textAlignment,
+                                                           positionController: positionController,
+                                                           tokenSet: tokenSet)
+        if let tooltipViewController = tooltipViewController,
```

Comment:
`
       if let tooltipViewController = tooltipViewController,
           let tooltipView = tooltipViewController.view,
           let rootViewController = window.rootViewController {
`

all these let. does it require If statement? curious if it is creating unnecessary indentation

Reply:
requires force unwrapping otherwise, but i'll do guard let instead as you suggested on Teams!